Automattic/woocommerce-payments
# Improve authorization capture notification message in payment details page

Fixes #5192 

#### Changes proposed in this Pull Request

Improve the capture notification section that appears in the payment details section when it is pending to be captured.
* Include a link to the capture section in public documentation.
* Improve the deadline message to indicate a relative date / time , to help the merchant understand the deadline/ urgency better . eg - `You need to capture this charge in 6 days`

#### Screenshots 

Message with the tooltip:

<img width="767" alt="Screenshot 2023-03-30 at 5 16 39 PM" src="https://user-images.githubusercontent.com/4162931/228826411-a1c09347-8809-4c48-8544-8a63f60f9d5d.png">

#### Screenshots with different duration

**Hours:** 
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/4162931/229019571-e081bc18-dae8-454b-9fbf-df2dea8c56fa.png">

**Minutes:**
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/4162931/229021949-11f06a19-89be-446b-9587-73f559b27f8b.png">

#### Testing instructions

* Within `Payments > Settings > Transactions` make sure to enable `Issue an authorization on checkout, and capture later` , and click the blue `Save Changes` button
* From the front end, create at least three to four orders 
* Browse to `Payments > Transactions > Uncaptured tab`
* Click on any uncaptured transactions to open its `Payment Details` page
* The yellow box with the notice should read in a format similar to the screenshot shown above
* If you mouseover for a few seconds, the tooltip should show the actual deadline date, as shown in the screenshot. 


-------------------


**Post merge**
